website/src/content/docs/docs/extending-typespec/emitter-framework.md

```diff
@@ -3,369 +3,189 @@ id: emitter-framework
+2. **alloy language components**: Libraries of components for various languages. For example, allows for declaring a variable in TypeScript, and then creating a reference to that variable such that any necessary imports and package.json dependencies are added. [Read the documentation for these components](https://alloy-framework.github.io/alloy/guides/getting-started/).
+3. **typekits**: A convenient way to pull information from the TypeSpec type graph.
+4. **emitter framework**: builds off of alloy and its language components to provide TypeSpec-aware components and utilities. For example, while the alloy language component for TypeScript interfaces allows manually declaring members, the emitter framework component can take a TypeSpec data type and convert it directly into a TypeScript interface.
 
-  // lastly, write your emit output into the output directory
-  await assetEmitter.writeOutput();
-}
-```
+When you are writing a TypeSpec emitter, you will draw on components and APIs from each of these areas.
 
-### `TypeEmitter<T>`
+## How to get started
 
-This is the base class for writing logic to convert TypeSpec types into assets in your target language. Every TypeSpec type has at least one method on this base class, and many have multiple methods. For example, models have both `ModelDeclaration` and `ModelLiteral` methods to handle `model Pet { }` declarations and `{ anonymous: boolean }` literals respectively.
+We recommend you read the alloy documentation to understand its component model, reactivity system, formatting capabilities, and so forth. Alloy's [core concepts](https://alloy-framework.github.io/alloy/guides/basic-concepts/) document is a good place to start.
 
-To support emitting all TypeSpec types, you should expect to implement all of these methods. But if you don't want to support emitting all TypeSpec types, you can either throw or just not implement the method, in which case the type will not be emitted.
+Also, make sure to read the 'Getting Started' section under the [emitter basics](./emitters-basics.md) topic. To use the framework, you will need an emitter library and a `$onEmit` function.
 
-The generic type parameter `T` is the type of emit output you are building. For example, if you're emitting source code, `T` will be `string`. If you're building an object graph like JSON, `T` will be `object`. If your `T` is `string`, i.e. you are building source code, you will probably want to use the `CodeTypeEmitter` subclass which is a bit more convenient, but `TypeEmitter<string>` will also work fine.
+## Add necessary dependencies
 
-Here's a simple emitter that doesn't do much yet:
+There are a few dependencies needed to use the emitter framework:
 
-```typescript
-class MyCodeEmitter extends CodeTypeEmitter {
-  modelDeclaration(model: Model, name: string) {
-    console.log("Emitting a model named", name);
-  }
-}
+```sh
+install > npm --save-peer @alloy-js/core @alloy-js/typescript @typespec/emitter-framework
```

Comment:
```suggestion
npm install --save-peer @alloy-js/core @alloy-js/typescript @typespec/emitter-framework
```

Reply:
what a weird error here

Reply:
I think its when you write 
```
> foo bar
```
in a bash code block the formatter thinks you meant to pipe later or something like that

Reply:
Yeah  - looks like the formatter did it again with the `>` in there.

I think `>` should probably just be ommitted (that way the `copy` button works too!)